### ISSUE TITLE
My remote IR of my Epson videoprojector and my onkyo Home cinema aren't detected

### ISSUE THREAD

**Issue by diazovitch** [Whilser/Broadlink-RM2-Universal-IR-Remote-Controller-Domoticz-plugin#2]
Hi,

Thank you for your plugin.
No problem for install.
I recorded my tv remote but i can't with my Epson videoprojector remote and my onkyo Home cinema remote ?
Could you explain this ?

**Comment by diazovitch** [comment]
And How you can clear all data recorded ?

**Comment by Whilser** [comment]
Hi! Yes, some remotes can transmit more information and the device may not fully capture the signal. Usually these are some models of air conditioners. In order to completely clear the created devices, you need to delete the json file in the plugin folder and restart the plugin.

**Comment by diazovitch** [comment]
Thank you for the quick reply.
Before I used another plugin that knew how to recognize these remotes.
http://domo-attitude.fr/broadlink-rm-promini-domoticz/?fbclid=IwAR2hQya-fQ9UKzHVE5gZp44SXX4iAgx1cJpvvbWkX-yMxCNqVEIz2mLT0j4

But there is a problem because, I can't recorded more one button.

**Comment by diazovitch** [comment]
After checking, it can register all my remotes but only the first IR. Cannot save 2 frames.

**Comment by Whilser** [comment]
What do you mean? To add a switch Press `Reset Level -> Learn` after the indicator on the Broadlink IR Remote Controller blinked, point the remote at it and press the button that you want to add to turn on, then press `Test` and if everything went well and command worked - press `Save Level`. To capture the next command, press Learn after the indicator on the Broadlink IR Remote Controller has blinked, press the button that you want to add to shutdown, then press Test. If the command worked correctly, click `Save Level`. After adding the command and successful testing, click `Create` to create the switch. If for some reason the command is not recognized, just press Reset Level and repeat everything from the beginning.

Adding a selector switch is no different, except that the procedure for scanning the IR code must be repeated as many times as necessary levels, pressing each time after adding the `Save Level` level, after all the commands have been added, click `Create`.

**Comment by diazovitch** [comment]
I do it but i wan't to recorded the second each time with all remotes.
I try without do the test and it seems to be working just for the first switch.

**Comment by diazovitch** [comment]
{
    "Unit 2": [
        {
            "LearnedCodes": {
                "IRCode1": "2600f80100012892143515341411151114111312141114351435141214341412143514121335141214111312$
            },
            "Level": 10
        },
        {
            "LearnedCodes": {
                "IRCode1": "2600200100012991143514361312141113131312131213351435141214351411153414121335141214351312$
            },
            "Level": 20
        }
    ],
    "Unit 3": [
        {
            "LearnedCodes": {
                "IRCode1": "000000000000000000000000"
            },
            "Level": 10
        }
    ],
    "Unit 4": [
        {
            "LearnedCodes": {
                "IRCode1": "000000000000000000000000"
            },
            "Level": 10
        }
    ],
    "Unit 5": [
        {
            "LearnedCodes": {
                "IRCode1": "000000000000000000000000"
            },
            "Level": 10
        },
        {

**Comment by Whilser** [comment]
Yes, it looks like other units are not captured for some reason.

**Comment by diazovitch** [comment]
I use raspbian buster and domoticz v2020.2 Do you test on this ?

**Comment by Whilser** [comment]
This plugin was tested with python 3.5.x and Domoticz 4.x installed on Raspberry Pi stretch. Unfortunately, I cannot test it on last software versions because I don't use domoticz anymore.

**Comment by diazovitch** [comment]
Too bad for me. Since this domoticz 2020.2 version, I have not found any effective solution for the broadlink. Yours looked promising.
